eng/build.yml

```diff
@@ -15,7 +15,7 @@ parameters:
   timeoutInMinutes: 180
 
 jobs:
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/6.0.1xx')) }}:
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/6.0.2xx')) }}:
```

Comment:
Who's responsible for keeping this up-to-date?  Should it be part of a branch creation checklist or something?

Reply:
I believe that's what msbuild does and I recommend that approach for sure.

Reply:
I've added it to our branch creation onenote.